src/ApiHandler.ts

```diff
@@ -124,20 +130,91 @@ export default class ApiHandler {
 			}
 		}
 
+		const fixed128 = new BN(10).pow(new BN(18))
+		const perBill = new BN(1e9);
+		const perByteFee = api.consts.transactionPayment.transactionByteFee;
+		const extrinsicBaseWeight = api.consts.system.extrinsicBaseWeight;
+		// Here we assume that `weightToFee` Vec only has 1 element, which is of
+		// degree 1. FIXME Implement polynomial.
+		const coefficients = api.consts.transactionPayment.weightToFee[0];
```

Comment:
I'm getting `Internal Error: TypeError: Cannot read property '0' of undefined` here. Not latest binary, but latest block.

Reply:
This field is a recent addition. You need to update your node.

Reply:
@athei still having the same issue on stable Kusama 0.7.33 release, fully synced.

Reply:
The change is not tagged yet. You need to build master.